lxd/storage/drivers/driver_ceph.go

```diff
@@ -112,6 +114,29 @@ func (d *ceph) FillConfig() error {
 		d.config["ceph.osd.pg_num"] = "32"
 	}
 
+	if d.config["ceph.osd.pool_size"] == "" {
+		size, err := shared.TryRunCommand("ceph",
+			"--name", fmt.Sprintf("client.%s", d.config["ceph.user.name"]),
```

Comment:
It's a tiny nit but simple string concat is more efficient and needs less keystrokes ;)
```suggestion
			"--name", "client." + d.config["ceph.user.name"],
```

Reply:
Someone was paying attention at the performance sessions in Madrid :)

Fixed

Reply:
Indeed!